README.md

```diff
@@ -54,7 +54,11 @@ ngx.ctx.authenticated_consumer = {
 
 ### XMLHttp/Ajax Requests
 
-HTTP Requests made by client-side code (e.g ajax) should include the `X-Requested-With: XMLHttpRequest` header. 302 Redirects are replaced with 401 Unauthorized HTTP responses when this header is present AND the user is unauthenticated.
+XMLHttpRequests made by client-side code (i.e ajax) should include the `X-Requested-With: XMLHttpRequest` header. 302 Redirects are replaced with 401 Unauthorized HTTP responses when this header is present AND the user is unauthenticated.
+
+#### Why?
+
+302 redirects are followed transparently via XMLHttpRequests (xhr/ajax requests) thus there is nothing the client side can do to detect if a 302 happened. Returning a status code of 401 allows the client to respond to the request accordingly.
```

Comment:
In Claris, we include a body with the 401 that contains the redirect location - can / should we do that here, so that the caller has the option of requesting the redirected url?